src/Uno.UI/UI/Xaml/Controls/Flyout/FlyoutPresenter.cs

```diff
@@ -8,36 +8,8 @@ public FlyoutPresenter()
 		{
 		}
 
-		protected override void OnPointerPressed(PointerRoutedEventArgs args)
```

Comment:
Can you explain why this code is no longer needed/correct?

Also worth checking with @dr1rrb if these changes will make sense once his pointer events feature branch is merged.

Reply:
Because they are duplicated in some other files, I also removed them in my PR